docs/source/time-correlation/time-correlation.rst

```diff
@@ -48,7 +56,7 @@ where :math:`\mathcal{F}\left[ \cdot \right]` is the Fourier transform of a func
 Function Documentation
 -----------------------
 
-.. autofunction:: tcf.tcf.autocorrelate
+.. automodule:: tcf.tcf.autocorrelate
    :noindex:
 
```

Comment:
Explain the normalization in your example and add a function showing that the explicit calculation that goes as O(N^2) gives the same output